spec/features/repositories/create_repository_spec.rb

```diff
@@ -152,7 +153,9 @@
 
         find('button[type="submit"]', text: I18n.t(:button_create)).click
 
-        expect(page).to have_selector('button[type="submit"]', text: I18n.t(:button_save))
+        expect(page).to have_selector('button[type="submit"]',
+                                      text: I18n.t(:button_save),
+                                      wait: 10)
```

Comment:
Just a gut feeling, but I would still try to add an expectation for the flash message before I try to guess the right amount of wait time. 

I believe that selenium/capybara has problems here because the element you query for is there before and after the reloading (or an element with the same selector) and thus it might change while capybara is interacting with it.

When adding a safeguard in the form of an expectation on the flash message, an element that has not been there before, this should no longer be a problem.